### ISSUE TITLE
allow updating checkout[note]?

### ISSUE THREAD

**Issue by hgoldwire** [adearriba/ShopifyCheckoutJS#2]
Thanks for making this lib! I really like the ergonomics and I'm grateful for all the work you saved me. I was doing a bad job of making this myself 😄 

One thing I'd like to do is edit checkout[note] to add a gift note, if the customer forgot to do so in cart, or if they bypassed cart and went straight to checkout.

It seems every field we add with this lib becomes checkout[attribute][whatever].  Any opinions about allowing an override for the field name?

Thanks!

**Comment by adearriba** [comment]
Hi @hgoldwire , thanks for your feedback! Much appreciated! 

For the fields, the structure is checkout[attribute], because this is the way Shopify works for custom attribute values. This is the structure Shopify uses to save that data inside of the order.

For notes, I will have to validate how to configure the note field so that the input is stored inside the notes of the order. Right now you can create a textfield and the value will be saved inside of the order attributes.

Best,
Alejandro.

**Comment by adearriba** [comment]
You are correct, seems "checkout[note]" works. I will add this new type of field tomorrow and push a new build also.
Thanks for the suggestion @hgoldwire.
